plugins/modules/inventory_workflow_manager.py

```diff
@@ -1619,7 +1629,9 @@ def handle_all_provisioned(self, device_type):
 
         self.status = "success"
         self.result['changed'] = True
-        self.log("All {0} Devices provisioned successfully!!".format(device_type), "INFO")
+        self.msg = ("All {0} Devices provisioned successfully!!".format(device_type), "INFO")
```

Comment:
Every time when you are assigning some value in the self.msg it gets overwritten in next steps and as a result we will missed the message in the console.

Reply:
self.msg is required for the check return status